cpp/modmesh/mesh/StaticMesh.hpp

```diff
@@ -112,12 +111,13 @@ struct CellType : NumberBase<int32_t, double>
     }
 
 private:
+    static constexpr const uint8_t NNODE_IDX = 1;
+    static constexpr const uint8_t NEDGE_IDX = 2;
+    static constexpr const uint8_t NSURFACE_IDX = 0;
 
     uint8_t m_id : 6;
     uint8_t m_ndim : 2;
-    uint8_t m_nnode = 0;
-    uint8_t m_nedge = 0;
-    uint8_t m_nsurface = 0;
+    uint8_t m_attrs[3] = {0};
```

Comment:
Why `= {0}` rather than `= {0, 0, 0}`?  Invalid syntax?

If it's valid, the latter reads more explicit and less error-prone.